mobilecoind/src/payments.rs

```diff
@@ -848,6 +856,7 @@ impl<T: BlockchainConnection + UserTxConnection + 'static, FPR: FogPubkeyResolve
         tombstone_block: BlockIndex,
         fog_resolver_factory: &Arc<dyn Fn(&[FogUri]) -> Result<FPR, String> + Send + Sync>,
         opt_memo_builder: Option<Box<dyn MemoBuilder + 'static + Send + Sync>>,
+        fee_map: FeeMap,
```

Comment:
:wrench: Suggest updating the doc string for this argument, since the other ones are in it.